doc/development/plugins.rst

```diff
@@ -97,169 +92,165 @@ as well as potential further capabilities, by providing the following class attr
 
     operations = {"CNOT", "PauliX"}
 
-  This is used to decide whether an operation is supported by your device in the
-  default implementation of the public method :meth:`.Device.supports_operation`.
-
-* :attr:`.Device.observables`: set of the supported PennyLane observables as strings, e.g.,
-
-  .. code-block:: python
-
-    observables = {"PauliX", "PauliZ", "Hadamard", "Hermitian"}
-
-  This is used to decide whether an observable is supported by your device in the
-  default implementation of the public method :meth:`.Device.supports_observable`.
```

Comment:
Cool that these are now supported by default! :blush:

Reply:
Yes, while writing this I realised that _every_ device that inherits from `QubitDevice` will support all observables automatically, so might as well add them by default.